compute-service/lambda/functions/helper_functions.py

```diff
@@ -1,38 +1,36 @@
-#import library
+# import library
 import re
 
-#function to check regex pattern
 
 def check_and_replace_regex(df, column_name, regex_pattern, replacement_value):
     """
-    Checks if values in a DataFrame column match a regex pattern. Replaces non-matching values.
-    
+    Checks if values in a DataFrame column match a regex pattern.
+    Replaces non-matching values.
     Parameters:
     - df (pd.DataFrame): The DataFrame to process.
```

Comment:
This is not how to define your parameters within function inside a Doc string
Use this as a ref https://github.com/cappymayor/random-profile-ingestion/blob/master/airflow/dags/utils/random_user_logics.py#L80-L86